### PR TITLE
escape quote chars even when --compress option is not used

### PR DESCRIPTION
**Pull request by GabrielDelepine**
Quotes escape was apply only if --compress was used.
This PR apply the quote escaping with or without using --compress

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -82,12 +82,13 @@ function processTemplateUrl(content, options, targetDir) {
 				ignoreCustomFragments: [/\s\[.*\]=\"[^\"]*\"/, /\s\([^)"]+\)=\"[^\"]*\"/]
 			});
```
Comment (pablodenadai) [inline index.js]: Remove empty line.

File: index.js [hunk 2]
```diff
@@ -82,12 +82,13 @@ function processTemplateUrl(content, options, targetDir) {
...
 
-			// escape quote chars
-			file = file.replace(new RegExp(quote, 'g'), '\\' + quote);
 		} else {
 			file = file.replace(/[\r\n]\s*/g, '');
 		}
 
+		// escape quote chars
+		file = file.replace(new RegExp(quote, 'g'), '\\' + quote);
+
```
Comment (ghpabs) [inline index.js]: Delete empty line

**Comment by pablodenadai** [comment]
Thanks mate - just gotta get the build passing now.

**Comment by GabrielDelepine** [comment]
Thanks, you are really fast !

**Comment by pablodenadai** [comment]
Do you need a new version on NPM?

**Pull request merged**